app/sql/schema.sql

```diff
@@ -63,6 +64,17 @@ CREATE TABLE IF NOT EXISTS `Product` (
     )
 );
 
+CREATE TABLE IF NOT EXISTS `ProductStatus` (
```

Comment:
Should be replaced with a boolean "listed" and moved into the Product table, unless we see a reason for having more than 2 types of statuses.